sections/06-vibrant-bodies.md

```diff
@@ -0,0 +1,201 @@
+
+Notably, Voyce argues that, underneath the agonistic and oppositional tropes of modernism, open community tactics have been an ongoing feature of avant-garde art movements such as Dada and Fluxus. For Voyce, this method of inter-authorial materiality suggests that avant-garde artists have an ongoing role and responsibility "to create and fortify public domains of open source knowledge, to challenge excessive restrictions placed on language and information, to bring forth marginalized knowledges from a position of inaccessibility to the public at large, and to produce and share artistic tactics and works that challenge intellectual property. That which is at stake is nothing less than open accessibility to culture" (427-8).
+
+## Material Systems: Font Màgica, Chris Welsby, and Tofino
+
+### Font Màgica
+
+For my soundtrack to Izabella Pruska-Oldenhof's film _Font Màgica_, at her request, I used a recording of Chopin's Prelude No. 15 ("Raindrop") as source material, layering and blending it with the sound of radio astronomy events. Since Izabella had initially edited the film to the Chopin piece, I wanted to retain some of the affective logic of Rudolf Serkin's expressive piano performance in my treatment of it. Using a network of amplitude trackers and granular signal processors, the piece's formal structure, its dynamic phrasing, was used as a signal with which to modulate and filter the playback of multiple layers of the Chopin along with the astronomical sounds. The Raindrop prelude's own compositional structure was fed back into the system, refracting itself into multiple layers of sound that are eerily offset from and mis-registered to the original, but which nonetheless carry a strangely warped expressivity. Like Izabella's visuals, it is an overtly refractive piece, full of prismatic colours, distorted ghost images, and passing fragments of a receding original.
+
+_Font Màgica_ gently sounds within a technics and phenomenology of material transformation. Neither a work _of_ traditionally-constructed originality nor one _about_ appropriation, it is a lensing apparatus, a technical-sonic system that is _complicit with_ Chopin and solar flares and Izabella's visuals and my signal processing algorithms. Though slightly different each time due to random signal changes, _Font Màgica_ is not an "open" work so much as an obsessed one. It is obsessed with its own material closing in on itself via signal processing. Chopin's _Raindrop_ appears everywhere, replicated at all levels as both a structuring force as well as a microsonic substrate. _Font Màgica_ aims for a form of systemic closure and inner self-similarity; a closure that paradoxically "realizes openness in its radical sense: not as openness toward the possibility of contingencies from the outside, but as a 'being opened' by the contingent materials that form the work" (Negarestani 3).
```

Comment:
Comment from David: "Why is it within a ‘technics and phenomenology of material transformation’? I’m not sure i understand that sentence (though I understand what follows). And do you really mean both? i.e. should the ‘and’ be italicized?

Further to that, I wonder if it’s worth checking instances of technics to see if there are some cases where you really mean techniques."